### ISSUE TITLE
Multiple files conflicts issue

### ISSUE THREAD

**Issue by pratibha77118** [ubsuny/23-Homework4G5#66]
Hi @tirthbha @uarif ,
I am confused if there are multiple .py files in the same directory (for e.g. i n mains), does the git hub actions conflicts while running? I think we  need to do the pytest and pylint in seperate branch to avoid the conflits. Also the workflow shows multiple actions running at the same time, how to avoid this?

**Comment by uarif** [comment]
@pratibha77118 The code is the one with plot....py.

**Comment by uarif** [comment]
I think it should not conflict. I believe it should work only for the file which is mentioned in that .yml file @pratibha77118

**Comment by tirthbha** [comment]
@pratibha77118 Yes, @uarif is right. In fact, the workflow will run for the particular .py file that you allow to run. Conflicts  are might be due to the same name that you assigned to the workflow.
